lib/hooks/submission-meta/index.js

```diff
@@ -0,0 +1,26 @@
+const { get } = require('lodash');
+
+module.exports = settings => {
+  const { Certificate, Exemption } = settings.models;
+
+  return model => {
+    const type = get(model, 'data.model');
+    const action = get(model, 'data.action');
+
+    if (type !== 'pil') {
+      return Promise.resolve();
+    }
+
+    if (action !== 'grant' && action !== 'transfer') {
```

Comment:
What are the things we're trying to ignore here? All I can think of is reviews, and it seems like perhaps they should have training attached too.

Reply:
yeah, I thought that, then I noticed we don't show training for reviews in task views, I'm not sure _why_, perhaps for this exact reason. maybe we'll never know.

Reply:
I feel like it batter to have the data and not need it, than to need it and not have it, so I'd probably dispense with this check and add the metadata to all PIL related tasks.

Reply:
If there are some we _know_ we don't need (create?) then potentially filter those out.